### ISSUE TITLE
unable to run grunt

### ISSUE THREAD

**Issue by rosskevin** [wnr/element-resize-detector#72]
Attempting to help out with #71, `npm i && npm run build`, and cannot run it.  I'm in unfamiliar territory, but it seems like the grunt-karma version specified is over two years old.

```
❯❯❯ ./node_modules/.bin/grunt --verbose                                                                                                                                                                ⏎ babili-fix
Initializing
Command-line options: --verbose

Reading "Gruntfile.js" Gruntfile...OK

Registering Gruntfile tasks.

Registering "grunt-banner" local Npm module tasks.
Reading /Users/kross/projects/element-resize-detector/node_modules/grunt-banner/package.json...OK
Parsing /Users/kross/projects/element-resize-detector/node_modules/grunt-banner/package.json...OK
Loading "usebanner.js" tasks...OK
+ usebanner

Registering "grunt-browserify" local Npm module tasks.
Reading /Users/kross/projects/element-resize-detector/node_modules/grunt-browserify/package.json...OK
Parsing /Users/kross/projects/element-resize-detector/node_modules/grunt-browserify/package.json...OK
Loading "browserify.js" tasks...OK
+ browserify

Registering "grunt-contrib-jshint" local Npm module tasks.
Reading /Users/kross/projects/element-resize-detector/node_modules/grunt-contrib-jshint/package.json...OK
Parsing /Users/kross/projects/element-resize-detector/node_modules/grunt-contrib-jshint/package.json...OK
Loading "jshint.js" tasks...OK
+ jshint

Registering "grunt-contrib-uglify" local Npm module tasks.
Reading /Users/kross/projects/element-resize-detector/node_modules/grunt-contrib-uglify/package.json...OK
Parsing /Users/kross/projects/element-resize-detector/node_modules/grunt-contrib-uglify/package.json...OK
Loading "uglify.js" tasks...OK
+ uglify

Registering "grunt-karma" local Npm module tasks.
Reading /Users/kross/projects/element-resize-detector/node_modules/grunt-karma/package.json...OK
Parsing /Users/kross/projects/element-resize-detector/node_modules/grunt-karma/package.json...OK
Loading "grunt-karma.js" tasks...ERROR
>> TypeError: Cannot read property 'prototype' of undefined
>>     at Object.<anonymous> (/Users/kross/projects/element-resize-detector/node_modules/socket.io/lib/store.js:35:41)
>>     at Module._compile (module.js:571:32)
>>     at Object.Module._extensions..js (module.js:580:10)
>>     at Module.load (module.js:488:32)
>>     at tryModuleLoad (module.js:447:12)
>>     at Function.Module._load (module.js:439:3)
>>     at Module.require (module.js:498:17)
>>     at require (internal/module.js:20:19)
>>     at Object.<anonymous> (/Users/kross/projects/element-resize-detector/node_modules/socket.io/lib/manager.js:16:13)
>>     at Module._compile (module.js:571:32)
Reading package.json...OK
Parsing package.json...OK
Initializing config...OK
Loading "Gruntfile.js" tasks...OK
+ build, build:dev, build:dist, ci, default, dist, sauceConnectTunnel, test, test:sauce, test:style

No tasks specified, running default tasks.
Running tasks: default

Running "default" task

Running "test" task
Warning: Task "karma:local" not found. Use --force to continue.

Aborted due to warnings.
```

**Comment by wnr** [comment]
All dependencies have now been updated, check it out in branch https://github.com/wnr/element-resize-detector/tree/WIP-1.1.11

**Comment by wnr** [comment]
Closing this, as this has been fixed inthe 1.1.11 WIP branch.
